plugin/NERD_commenter.vim

```diff
@@ -204,6 +204,7 @@ let s:delimiterMap = {
     \ 'hbs': { 'left': '{{!-- ', 'right': ' --}}' },
     \ 'hercules': { 'left': '//', 'leftAlt': '/*', 'rightAlt': '*/' },
     \ 'hive': { 'left': '-- ' },
+    \ 'hocon': { 'left': '#' },
```

Comment:
Why use `#` as the default? The docs you linked to used `//` in most of their examples, it seems like that might be more idiomatic. Whichever one you use, you should also include a `leftAlt` key with the other one.

Reply:
No reason in particular, just Ruby mnemonics. I updated the code to default on `//`.